cumulusci/robotframework/tests/salesforce/retry.robot

```diff
@@ -0,0 +1,122 @@
+*** Settings ***
```

Comment:
Currently our CI for CumulusCI itself only runs specific robot test files, so you would need to add this one to the list in order for CircleCI to run it. https://github.com/SFDO-Tooling/CumulusCI/blob/master/.circleci/config.yml#L98